staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -663,3 +675,55 @@ func fakeVersion() version.Info {
 		GitTreeState: "Dirty",
 	}
 }
+
+// generateSelfSignedCertKey creates a self-signed certificate and key for the given host.
+// Host may be an IP or a DNS name
+// You may also specify additional subject alt names (either ip or dns names) for the certificate
+func generateSelfSignedCertKey(host string, alternateIPs []net.IP, alternateDNS []string) ([]byte, []byte, error) {
```

Comment:
Or modify `utilcert.GenerateSelfSignedCertKey` to take in a `ttl`? It's only used in 2 places in the entire code base.